lib/registries/addon/branded/route.ts

```diff
@@ -4,13 +4,19 @@ import { inject as service } from '@ember/service';
 import DS from 'ember-data';
 
 import Analytics from 'ember-osf-web/services/analytics';
+import Brand from 'registries/services/brand';
 
 export default class BrandedRegistriesRoute extends Route {
     @service analytics!: Analytics;
     @service store!: DS.Store;
+    @service brand!: Brand;
 
     model(params: { providerId: string }) {
-        return this.store.findRecord('registration-provider', params.providerId);
+        return this.store.findRecord('registration-provider', params.providerId, { include: 'brand' });
```

Comment:
@aaxelb For some reason fetching the brand this way doesn't work. When I access `registrationProvder.brand`, it evaluates to null. I know that in the fixture, brand is set to null, but I do associate the brand in the default scenario. Is there something I'm missing?

Reply:
Nm I figured it out. I had to add something in the registration-provider serializer.

Reply:
Although I'm running into a lot of Mirage errors because the relationship may not be set up correctly, or maybe the config isn't set up properly

Reply:
@robustdj checked it out locally -- i think the explicit `brand: null` in the `registration-providers` fixture may somehow mess up mirage's db so you can't actually update the brand. removing that might help fix?